osu.Game/Stores/RealmArchiveModelImporter.cs

```diff
@@ -389,7 +389,10 @@ protected virtual string ComputeHash(TModel item)
                                 LogForModel(item, @$"Found existing {HumanisedModelName} for {item} (ID {existing.ID}) – skipping import.");
 
                                 if (existing.DeletePending)
+                                {
+                                    LogForModel(item, $@"Existing {HumanisedModelName}'s deletion flag has been removed");
```

Comment:
I would fold this log statement into `UndeleteForReuse()`, because that method has more callers that may also benefit from this kind of logging.